src/locales/cs.json

```diff
@@ -965,6 +965,7 @@
   "app.randomMessages.m9": "Nakreslit hrocha",
   "app.referenceSolution.exerciseBroken": "Úloha je rozbitá, a proto není v tuto chvíli možné nechat toto vzorové řešení znovu vyhodnotit.",
   "app.referenceSolution.exerciseNoLongerHasEnvironment": "Úloha již nepodporuje běhové prostředí, pod kterým bylo toto řešení odevzdáno. Řešení není možné znovu vyhodnotit.",
+  "app.referenceSolution.reevaluatedBy": "Nechal znovu vyhodnotit",
```

Comment:
This is not gender-balanced. At least write 'Nechal(a)', or rewrite to something like 'Znovu vyhodnoceno uzivatelem'.

Reply:
Again... copied... will change

Reply:
OK, change the source as well, or better yet, merge the captions if they are really the same.